presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,63 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix accounting of time spent reading Parquet data.
+* Fix rare (but practical) incorrect query result for queries involving timestamp,
+  when the time zone database embedded in the version of Joda-Time that Presto depends on
+  does not match the time zone database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when optimize_mixed_distinct_aggregation is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add :func:`reduce_agg` aggregate function.
+* Add :func:`milliseconds` function (issue #9524).
+* Add support for predicate pushdown for partition symbols for window operators.
+* Remove SHOW STATS ON syntax
```

Comment:
maybe add that SHOW STATS FOR should be used instead

Reply:
also, it looks like we put SQL syntax in code backticks. so that should be done here too (or removed at show create table.  whichever is our standard practice)

Reply:
Yes please put code, SQL stuff in double backticks. There are other places that need to be updated.

Reply:
Also need a `.` at the end.